spec/support/ansible_shared/tower_data_helper.rb

```diff
@@ -0,0 +1,3 @@
+shared_context "uses tower_data.yml" do
+  let(:tower_data) { YAML.load_file(ManageIQ::Providers::AnsibleTower::Engine.root.join("spec/support/tower_data.yml")) }
```

Comment:
I still don’t like this. Can’t we turn the path into a constant then use it when writing the file by the Rake task?

https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/93/commits/7f4bdfeb147d0991b05db858cb5d5a7861f05fdb#diff-bb7272be93a294c53ae6433eb024074fR11

Reply:
Why would be a global constant better than a shared context? The context defines the test scope. Global constant would require the file to be loaded for any spec execution within this repo. A shared context allows lazy loading of the file - only if the spec requires it. In other words, this way it is not required to have the YAML present and loaded at all when you're running a spec which doesn't include the context.

Reply:
You just wait a _long_ time for nothing before going to the next step – populating item counts.

Reply:
There is nothing bad with the shared context. I’d just extract the file path to the global content. That is virtually no overhead (as compared to loading and parsing the whole file) for that and then in can be used even outside the specs for writing the file.